packages/hs-react-ui/src/context/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from 'react';
+import styled, { css, StyledComponentBase } from 'styled-components';
+import fonts from '../enums/fonts';
+import colorsEnum from '../enums/colors';
+
+export const defaultGlobalStyles = `
+  ${
+    process.env.NODE_ENV !== 'test'
```

Comment:
Are we polyfilling process.env so that this doesn't throw an error when running in a browser not through storybook? I'm only asking because I'm not sure how storybook rendering works or if it does any server side rendering for us.

Reply:
I've updated the logic for just in case, but it should work regardless. :D